FE/src/components/AccountMenus/LoginMenu/index.jsx

```diff
@@ -5,17 +5,21 @@ import { useAppContext } from "../../../context";
 import { LoginDiv, UserImg } from "./styled-components";
 
 async function sendLoginInfo(data, logger) {
+	console.log(data);
 	if (data.username === "" || data.password === "") {
 		return console.log("Data is empty");
 	}
-	fetch("http://localhost:8086/todo/users/login", {
+
+	const headers = new Headers();
+	headers.append("Content-Type", "application/json");
+
+	await fetch("http://localhost:8086/todo/users/login", {
 		method: "POST",
-		headers: {
-			"Content-Type": "application/json",
-		},
+		mode: "no-cors",
+		headers: headers,
 		redirect: "follow",
 		referrerPolicy: "no-referrer",
-		body: JSON.stringify(data),
+		body: JSON.stringify(data)
```

Comment:
```suggestion
		body: data
```
Acho que nao precisa ter isto, uma operacao a mais que nao faz diferenca nenhuma